base/irrationals.jl

```diff
@@ -5,8 +5,9 @@
 """
     AbstractIrrational <: Real
 
-Number type representing an exact irrational value, which is automatically rounded to the correct precision in
-arithmetic operations with other numeric quantities.
+Number type representing an exact irrational value, which is automatically rounded to the "correct" precision in
+arithmetic operations with other numeric quantities. Even the simplest calculations, like -pi, make convert a
+number to Float64, making the result no longer irrational.
```

Comment:
```suggestion
arithmetic operations with other numeric quantities. Even the simplest calculations, like `-pi` or `2pi`, make
convert a number to Float64, making the result no longer irrational.
```

Reply:
Good suggestion, this is implied multiplication. I did a bit differently and dropped a redundant word.